homeassistant/components/config/__init__.py

```diff
@@ -65,11 +65,11 @@ def component_loaded(event):
 
     hass.bus.async_listen(EVENT_COMPONENT_LOADED, component_loaded)
 
-    tasks = [setup_panel(panel_name) for panel_name in SECTIONS]
+    tasks = [asyncio.create_task(setup_panel(panel_name)) for panel_name in SECTIONS]
```

Comment:
We should probably use `hass.async_create_task` to allow tracking the tasks during set up and shutdown. The only exception is when the task would block home assistant from completing setup phase.

https://github.com/home-assistant/core/blob/2d9eb251427e237f0d4dcc939a0b0d0a826a978e/homeassistant/core.py#L382

Reply:
It actually won't matter because it's awaited inside `async_setup` which is itself already tracked by `hass`

Reply:
My comment was in general. I haven't analyzed all the upstream code paths in all these changes. Maybe better to just be safe?

Reply:
IMO this PR is fine since it does what `asyncio.wait()` has been doing for us so far.

Moving to `hass.async_create_task()` should (again, IMO) be done in separate PRs since it could have side effects unrelated to this deprecation.